.pre-commit-config.yaml

```diff
@@ -0,0 +1,22 @@
+repos:
```

Comment:
@thecodingwizard this look okay to you?

Reply:
sure LOL (i don't really know what they do)